db/mongo_console.go

```diff
@@ -0,0 +1,26 @@
+package db
+
+import (
+	"github.com/Scalingo/cli/Godeps/_workspace/src/gopkg.in/errgo.v1" // "mysql2://" for ruby driver 'mysql2'
+	"github.com/Scalingo/cli/apps"
+)
+
+func MongoConsole(app string) error {
+
+	mongoURL, user, password, err := dbURL(app, "MONGO", []string{"mongodb://"})
+	if err != nil {
+		return errgo.Mask(err)
+	}
+
+	opts := apps.RunOpts{
+		App: app,
+		Cmd: []string{"mongo", "-u", user, "-p", password, mongoURL.Host + "/" + user},
+	}
+
+	err = apps.Run(opts)
+	if err != nil {
+		return errgo.Newf("Fail to run redis console: %v", err)
```

Comment:
Hi there, you're completely right, and it has been fixed there: https://github.com/Scalingo/cli/commit/26b9d8f20a5d013cc9e1305bbb8df40eff1419c8

Fixed in next release